tests/normalize/test_normalize.py

```diff
@@ -708,3 +709,32 @@ def assert_timestamp_data_type(load_storage: LoadStorage, data_type: TDataType)
     event_schema = load_storage.normalized_packages.load_schema(loads[0])
     # in raw normalize timestamp column must not be coerced to timestamp
     assert event_schema.get_table_columns("event")["timestamp"]["data_type"] == data_type
+
+
+def test_removal_of_normalizer_schema_section(raw_normalize: Normalize) -> None:
```

Comment:
so this test must be a little extended.
1. we remove evolove-columns-once
2. but we add `seen-data` to all tables that had data items (see shcema utils: has_table_seen_data)
this is a good test, needs to be upgraded